compiler/one-cmds/one-init

```diff
@@ -0,0 +1,262 @@
+    def __init__(self):
+        # key as the default section
+        self.default_key = '[DEFAULT]'
+
+        # NOTE: keys of sections always have '[' and ']'
+        # key: string, value: list
+        self.sections = {self.default_key: [], }
+
+        # TODO: support ':' delimiter
+        self.delimiter = '='
+        self.comment_prefixes=('#', ';')
+
+    # read and parse
+    def read(self, cfg_path):
+        lines = ''
+        with open(cfg_path, 'r') as f:
+            lines = f.readlines()
+
+        cur_section = self.sections[self.default_key]
+        for line in lines:
+            # To ease, remove trailing new line
+            line = line.rstrip('\n')
+            if line.lstrip().startswith('['):
+                self.sections[line] = []
+                cur_section = self.sections[line]
+            else:
+                # Handle the case where strip() is needed
+                # key's head empty character can be an indention. Keep as it is
+                # But value's tail empty character is not necessary. Remove it
+                # And whitespaces btw key/value are unnecessary
+                # src ' key   =   value '
+                # dst ' key=value'
+                if self.delimiter in line:
+                    words = line.split(self.delimiter, maxsplit=1)
+                    # words[0]: ' key   '   -> ' key'
+                    # words[1]: '   value ' -> 'value'
+                    words[0] = words[0].rstrip()
+                    words[1] = words[1].strip()
+                    line = words[0] + self.delimiter + words[1]
+                cur_section.append(line)
+
+    def write(self, cfg_path):
+        # key: string, value: string
+        sections = {}
+
+        # convert first,
+        for section in self.sections.keys():
+            content = '\n'.join(self.sections[section])
+            sections[section] = content
+
+        with open(cfg_path, 'w') as f:
+            for k in sections.keys():
+                if k != self.default_key:
+                    f.write(k)
+                    f.write('\n')
+                f.write(sections[k])
+                f.write('\n')
+
+    # NOTE: section always start with '[' and end with ']'
+    def set(self, section, key, value):
+        # TODO: Handle exceptions for none of section and key
+        lines = self.sections[section]
+        for index, line in enumerate(lines):
+            # case: comments
+            if line.lstrip().startswith(self.comment_prefixes):
+                continue
+            # case: not strict config
+            if not self.delimiter in line:
+                continue
+
+            words = line.split(self.delimiter, maxsplit=1)
+            if key.strip() == words[0].strip():
+                lines[index] = words[0] + self.delimiter + value
+                return
+
+
+def _get_supported_map():
+    return {
+        'pb': 'tf',
+        'tflite': 'tflite',
+        'bcq': 'bcq',
+        'onnx': 'onnx',
+    }
+
```

Comment:
@YongseopKim , it is not mandatory for this PR, but I think we should provide an option to override this map. That is, if you give `--tflite` option along with `model.pb` and do `onecc init --tflite model.pb`, the model is regarded as a tflite model regardless of the extension `*.pb`. Because we cannot guarantee all models by extension alone.

Reply:
That's a great idea. This can be applied soon.